content/contents.lr

```diff
@@ -18,4 +18,4 @@ We promote interest in all areas of Computer Science, and host social events and
 
 - [Facebook group](https://www.facebook.com/groups/2202189009/)
 - [Mailing list](mailing_list)
-- [Slack](https://muncssociety.slack.com)
+- [Discord](https://discord.muncssociety.com)
```

Comment:
Domain name is muncompsci.ca

Reply:
3ac1c545e6a20ff3574be86855797f3b6117c821

Reply:
Nope, still wrong

Reply:
648374bc6717a229849c6911603ce3219fe038c3